src/audio/pipeline/chain_dma.c

```diff
@@ -0,0 +1,107 @@
+// SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Since this is a processing component/module (since it's using the comp APIs) it would be best to be placed with the others (i.e. not in core pipeline directory).